guide.html

```diff
@@ -688,18 +688,28 @@
           definition, that would be a problematic approach since it would risk linking the same term
           used with a different meaning. What it does instead is that it allows you to use an
           <code>href</code>-less <code>a</code> element around a term when you use it, and it will
-          manage the link on its own (e.g. <code>&lt;a>term&lt;/a></code>).
+          manage the link on its own (e.g., <code>&lt;a>term&lt;/a></code>).
         </p>
         <p>
           On occasion that won't be enough since you may be using the term in plural or conjugated,
           or in some other variant that does not exactly match the <code>dfn</code>. If so, then you
-          can use the <code>title</code> attribute on the <code>a</code> element in order to
-          indicate the exact term that you wish to have a link to.
+          can use the <code>lt</code> attribute on the <code>a</code> element in order to
+          indicate the exact term that you wish to have a link to (the mnemonic here being "link term").
+          Alternately, and more interestingly, 
+          you can specify an <code>lt</code> attribute on the associated <code>dfn</code> element with 
```

Comment:
I had neglected to do the build when I started the pull request.  I did it
in gh-pages.  It is all better now.


On Fri, Jul 31, 2015 at 10:48 AM, Marcos Caceres <notifications@github.com>
wrote:

> In guide.html
> <https://github.com/w3c/respec-docs/pull/33#discussion_r35986161>:
>
> >          </p>
> >          <p>
> >            On occasion that won't be enough since you may be using the term in plural or conjugated,
> >            or in some other variant that does not exactly match the <code>dfn</code>. If so, then you
> > -          can use the <code>title</code> attribute on the <code>a</code> element in order to
> > -          indicate the exact term that you wish to have a link to.
> > +          can use the <code>lt</code> attribute on the <code>a</code> element in order to
> > +          indicate the exact term that you wish to have a link to (the mnemonic here being "link term").
> > +          Alternately, and more interestingly,
> > +          you can specify an <code>lt</code> attribute on the associated <code>dfn</code> element with
>
> data-lt
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/respec-docs/pull/33/files#r35986161>.
>



-- 
Shane McCarron
halindrome@gmail.com